### PR TITLE
[Term Entry] Python:NumPy Math Methods log #4709

### PR DESCRIPTION
**Pull request by unicorns404**
### Description

- Added the log term math method for python NumPy
- Closes issue #4709 

### Type of Change
- Added new entry

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/numpy/concepts/math-methods/terms/log/log.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+```shell
+[0.         1.         2.30258509 4.60517019]
+```
+
+> **Note:** Running the above code will result in a `RuntimeWarning` due to attempting to calculate the logarithm of a negative number, which is not a valid operation.
+
+## Codebyte Example
+
+In this codebyte example, the `.log10()` method only computes the natural logarithm of positive elements in the array:
```
Comment (unicorns404) [inline content/numpy/concepts/math-methods/terms/log/log.md:53]: Change to '.log10()' to '.log()'

File: content/numpy/concepts/math-methods/terms/log/log.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+print(result)
+```
+
+The code above will generate the following output:
+
+```shell
+[0.         1.         2.30258509 4.60517019]
+```
+
+> **Note:** Running the above code will result in a `RuntimeWarning` due to attempting to calculate the logarithm of a negative number, which is not a valid operation.
```
Comment (unicorns404) [inline content/numpy/concepts/math-methods/terms/log/log.md:49]: Remove

File: content/numpy/concepts/math-methods/terms/log/log.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+```pseudo
+numpy.log(array, out=None, where=True)
+```
+
+- `array`: An array-like structure containing the elements for which the base-10 algorithm will be applied.
+- `out` (Optional): The array where the result is to be stored. If not provided, a new array is created to store the results.
+- `where` (Optional): The condition (array of boolean values) that determines which elements the method will be applied on.
+  - If the condition is `True` for a particular element, the logarithm is computed for that element.
+  - If the condition is `False` for a particular element, the logarithm is not computed for that element and the original element is retained.
+  - If not provided, the logarithm is computed for all elements.
```
Comment (mamtawardhani) [inline content/numpy/concepts/math-methods/terms/log/log.md:29]: This is directly plagiarised from: https://www.codecademy.com/resources/docs/numpy/math-methods/log10

Could you please reframe the sentences to solve this issue?

File: content/numpy/concepts/math-methods/terms/log/log.md (hunk 2 above)
Comment (mamtawardhani) [inline content/numpy/concepts/math-methods/terms/log/log.md:49]:
```suggestion
```

File: content/numpy/concepts/math-methods/terms/log/log.md (hunk 1 above)
Comment (mamtawardhani) [inline content/numpy/concepts/math-methods/terms/log/log.md:53]:
```suggestion
In this codebyte example, the `.log()` method only computes the natural logarithm of positive elements in the array:
```

**Review by mamtawardhani — CHANGES_REQUESTED** [review]
Thank you for contributing to the Codecademy Docs @unicorns404 😃 

I've left a comment and suggested come changes for this entry, could you please review and modify those at your earliest convenience? Feel free to dicuss any concerns, thank you! 😄

**Review by mamtawardhani — APPROVED** [review]
Thank you for the contribution @unicorns404 😄 

The entry looks good for a second round of review.

**Review by letsbuildanything — APPROVED** [review]
LGTM! 🚀

**Pull request merged**